apps/advisor/pkg/app/checks/datasourcecheck/check.go

```diff
@@ -235,7 +241,8 @@ func (s *missingPluginStep) Run(ctx context.Context, log logging.Logger, obj *ad
 				Url:     fmt.Sprintf("/connections/datasources/edit/%s", ds.UID),
 			},
 		}
-		_, err := s.PluginRepo.PluginInfo(ctx, ds.Type)
+		compatOpts := repo.NewCompatOpts(s.GrafanaVersion, sysruntime.GOOS, sysruntime.GOARCH)
```

Comment:
so this will be used for setting `User-Agent` header under the hood [here](https://github.com/grafana/grafana/pull/105122/files#diff-51f0dfa05bc316738bdabc58e916be8c522ca0e30ba00c3cf2b1ae85e2ab5aa8R143)?